ot/unbalanced.py

```diff
@@ -19,7 +19,8 @@
 from .utils import list_to_array, get_parameter_pair
 
 
-def sinkhorn_unbalanced(a, b, M, reg, reg_m, method='sinkhorn', numItermax=1000,
+def sinkhorn_unbalanced(a, b, M, reg, reg_m, reg_type="entropy", warmstart=None,
```

Comment:
add tye new parameter after method that should keep its sixth position (other parametrs should proably be apssed by name)